packages/repository/src/model.ts

```diff
@@ -47,6 +47,20 @@ export interface PropertyDefinition {
   [attribute: string]: any; // Other attributes
 }
 
+/**
+ * Defining the settings for a model
+ */
```

Comment:
I am proposing to add a link to https://loopback.io/doc/en/lb4/Model.html#supported-entries-of-model-definition where users can find more information about the supported model settings.

Reply:
Good idea, added